compiler/rustc_middle/src/query/mod.rs

```diff
@@ -1388,6 +1388,17 @@ rustc_queries! {
         desc { "computing layout of `{}`", key.value }
     }
 
+    /// Computes the naive layout estimate of a type. Note that this implicitly
+    /// executes in "reveal all" mode, and will normalize the input type.
+    ///
+    /// Unlike `layout_of`, this doesn't recurse behind reference types.
```

Comment:
explain this more. We're still figuring out whether a reference is a wide pointer or not, right?

Reply:
Yes, this still computes the layout of `<&T as Pointee>::Metadata`; I'll add a clarifying comment.